cli/mender-artifact/write.go

```diff
@@ -520,9 +534,8 @@ func getDeviceSnapshot(c *cli.Context) (string, error) {
 	// First echo to stdout such that we know when ssh connection is
 	// established (password prompt is written to /dev/tty directly,
 	// and hence impossible to detect).
-	// When user id is 0 do not bother with sudo.
```

Comment:
This was introduced explicitly to workaround [MEN-3987](https://tracker.mender.io/browse/MEN-3987), you should revert

Reply:
Thanks! Good catch.
This must've happened during rebase :-1: